providers/implementations/rands/drbg.c

```diff
@@ -446,18 +446,27 @@ int ossl_prov_drbg_instantiate(PROV_DRBG *drbg, unsigned int strength,
 
     if (!drbg->instantiate(drbg, entropy, entropylen, nonce, noncelen,
                            pers, perslen)) {
-        cleanup_entropy(drbg, entropy, entropylen);
         ERR_raise(ERR_LIB_PROV, PROV_R_ERROR_INSTANTIATING_DRBG);
         goto end;
     }
-    cleanup_entropy(drbg, entropy, entropylen);
-
+    l_prov_drbg_instantiate(PROV_DRBG *drbg, unsigned int strength,
```

Comment:
This isn't right, an unclosed call to l_prov_drbg_instatiate?  This looks more like a function prototype than a call.